sklearn/cross_decomposition/pls_.py

```diff
@@ -842,7 +864,7 @@ def fit_transform(self, X, y=None, **fit_params):
             Training vectors, where n_samples in the number of samples and
             p is the number of predictors.
 
-        Y : array-like of response, shape = [n_samples, q], optional
+        y : array-like of response, shape = [n_samples, q], optional
```

Comment:
Capitalizing it might break some user's code. Let's keep this PR simple.

I guess we can remove the `**fit_params` though.